lib/Config.js

```diff
@@ -662,6 +662,7 @@ class Config extends EventEmitter {
             this.pfsDaemon.dataPath =
                 process.env.PFSD_MOUNT_PATH ?
                 process.env.PFSD_MOUNT_PATH : `${__dirname}/../localPfs`;
+            this.pfsDaemon.isReadOnly = process.env.PFSD_READONLY || true;
```

Comment:
That would most likely not work as expected to allow writing, because `PFSD_READONLY` is a string coming straight from the environment, except maybe if an empty string it would be interpreted as truish.

To avoid the double-negation, i.e. asking the user to set a falsy env var to disable the read-only mode, I would recommend `PFSD_RDWR` or similar, so that the user can set it to any truish value, which is more common, i.e. enable write rather than disable no-write-allowed.

Reply:
Oh yeah, my bad. It should probably check if the string === 'true' and set the default to false.

However, I prefer to stick with `PFSD_READONLY` to match Kubernetes' NFS Volume spec `readOnly` attribute, since its connected to PFSD in the end.